docs/proposals/PROPOSAL_F.md

```diff
@@ -0,0 +1,190 @@
+      "kernel-selinux.enabled": true,
+      "kernel-version.major": 4           
+    },
+    "memory": {
+      "memory-numa": true,
+      "memory-nv.dax": true
+    },
+    "network": {
+      "network-sriov.capable": true,
+      "network-sriov.configured": true
+    }
+}
+```
+
+A similar spec for an HPC-oriented container might look like the following:
+
+```json
+{
+  "compatibilities": {
+    "cpu": {
+      "target": "arm64",
+    },
+    "mpi": {
+      "implementation": "openmpi",
+      "version": "4.1.6"           
+    },
+    "gpu": {
+      "enabled": true
+    },
+    "os": {
+      "vendor": "rocky",
+      "version": "8"
+    }
+}
+```
+
+As stated previously, the granularity of metadata provided is up to the compatibility interest group. The degree to which each metadata attribute is used is up to the plugin or tool within the context of a specific use case.
+
+## Requirements
+
+Taken from [REQUIREMENTS](../REQUIREMENTS.md). Points that have:
+
+- `TOOL RESPONSIBILITY` would be possible by a custom tool.
+- `AGNOSTIC`: the proposal is agnostic to the specific point (can be decided by working group)
+
+### Image Author
+
+- [x] As an image author, I want to update compatibility independently without having to re-release and re-distribute my image.
+- [x] As an image author, I want to have the freedom to express any compatibility that is necessary for my container to run on the host.
+  - [x] Including conditional compatibility, such as a container running on AMD CPU and intel CPU having different requirements.
+  - [x] Including must have/nice to have compatibility.
+  - [x]  Matching finer grain platform definitions.
+- [x] As an image author, I want to use a provided tool to verify the compatibility spec I wrote against the schema.
+- [ ] As an image author, I would like to create a single compatibility description that is common to a group of images (`TOOL RESPONSIBILITY`).
+- [ ] As an image author, I want to be able to specify in my manifest that my multi-platform image has a compatibility spec which should be consulted `AGNOSTIC`.
+- [ ]  As an image author, I want to be able to include compatibility specifications from base layers that I inherited from `TOOL RESPONSIBILITY` that can derive path of base layers
+- [x] As an image author, I want to ensure that runtimes without image compatibility gracefully fall back to running a usable image.
+
+### Domain Architect
+
+- [x] As a domain architect I want a process to share my knowledge about \<niche topic\> compatibility with some compatibility interest group.
+- [x] As a domain architect I don't want to have to understand containers or develop tools for them to share this knowledge.
+
+### Tool Writer
+
+- [x] As a tool writer, I want to get the compatibility spec without pulling the image layer blobs.
+- [x] As a tool writer, I would like to have a library for reading image compatibility so that I can write my own software that takes action based on the spec, e.g.:
```

Comment:
Is this creating/maintaining a library, or only the spec? Would it be a Go library similar to image-spec and distribution-spec?

Reply:
Ditto. I think this is a separate / next discussion.